_config.yml

```diff
@@ -21,6 +21,8 @@ description: >-
 url: "https://eips.ethereum.org"
 github_username:  ethereum
 
+incremental: true
+
```

Comment:
What does this do?

Reply:
This enables incremental build for quick testing. I forgot to remove it afterwards -- I'll fix that!